CMakeLists.txt

```diff
@@ -84,17 +86,16 @@ add_dependencies(jsonfortran json-fortran)
 set_property(TARGET jsonfortran PROPERTY IMPORTED_LOCATION ${BINARY_DIR}/lib/libjsonfortran.a)
```

Comment:
We want to build `jsonfortran` with spack and find it with pkconfig.

Reply:
This PR is not related to using Spack.